test/navigation/ObjectivesNavigatorSpec.scala

```diff
@@ -41,7 +43,21 @@ class ObjectivesNavigatorSpec extends SpecBase {
         "go to the CharitablePurposes page when clicked button" in {
           navigator.nextPage(CharitableObjectivesPage, NormalMode,
             emptyUserAnswers.set(CharitableObjectivesPage, "abcd").getOrElse(emptyUserAnswers)) mustBe
-            routes.IndexController.onPageLoad() // TODO modify once Charitable purposes page is created
+            operations.CharitablePurposesController.onPageLoad(NormalMode)
+        }
+      }
+
+      "from the CharitablePurposesPage" must {
+
+        "go to the SessionExpiredController page when user answer is empty" in {
+          navigator.nextPage(CharitablePurposesPage, NormalMode, emptyUserAnswers) mustBe
+            routes.SessionExpiredController.onPageLoad()
+        }
+
+        "go to the Index page when selected option other than Other and clicked continue" in {
```

Comment:
This title sohuld be updated to "go to the How does your charity benefit the public? page when selected option other than Other and clicked continue"